### PR TITLE
Update you-need-to-enable-secure-boot-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the title as the document pertains to checking the status of secure boot.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2743

**Comment by PRMerger6** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/user-help/you-need-to-enable-secure-boot-windows.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # required metadata
 
-title: Enable Secure Boot on Windows devices - Microsoft Intune | Microsoft Docs
+title: Check Secure Boot Status on Windows devices - Microsoft Intune | Microsoft Docs
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/you-need-to-enable-secure-boot-windows.md:4]:
```suggestion
title: Check Secure Boot status on Windows devices - Microsoft Intune | Microsoft Docs
```

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and signoff this PR. Thanks

**Review by lenewsad — CHANGES_REQUESTED** [review]
Hi @msbemba, thanks for looking into this issue. The content is about enabling secure boot but in the context of device enrollment. The preference is that people on enrolled devices contact their support person. The doc also mentions the option to enable secure boot on their own but since it's outside the scope of Intune, they need go to the Windows docs for that information. I think the SEO is really good and that's why it's coming out ahead of the Windows doc that isn't specific to enrollment. I may move this content to another article so that it's just a section in a larger document, so let's not change the title right now.

**Comment by yogkumgit** [comment]
@lenewsad Thanks for the comment. Closing

**Pull request closed without merging**